great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -204,11 +204,14 @@ def inner_wrapper(self, column, result_format = None, *args, **kwargs):
 
 class SqlAlchemyDataset(MetaSqlAlchemyDataset):
 
-    def __init__(self, table_name=None, engine=None, connection_string=None, custom_sql=None, *args, **kwargs):
+    def __init__(self, table_name=None, engine=None, connection_string=None,
+                 custom_sql=None, schema=None, *args, **kwargs):
         if table_name is None:
             raise ValueError("No table_name provided.")
 
         self.table_name = table_name
```

Comment:
Just a small question on this from reading through: do we still need the table_name property inside self anymore with this refactor?

Reply:
That's a good point. I don't think we'll need `table_name` or `schema` outside the initialization function. They should also be accessible from the `Table` object in any case.